src/Components/v2/Carousel.tsx

```diff
@@ -203,13 +229,11 @@ const CarouselContainer = styled.div<{ height?: number }>`
 export const ArrowButton = styled(Flex)<
   LeftProps & RightProps & { height?: number }
 >`
-  position: relative;
+  position: absolute;
```

Comment:
If you haven't already, can you please verify that these CSS changes still look good in all supported browsers?